src/store/reducers/stateCache/stateCache.types.ts

```diff
@@ -22,6 +22,29 @@ interface AuthenticationCacheProps {
 enum IncomingRequestType {
   CREDENTIAL_OFFER_RECEIVED = "credential-offer-received",
   MULTI_SIG_REQUEST_INCOMING = "multi-sig-request-incoming",
+  BALLOT_TRANSACTION_REQUEST = "ballot-transaction-request",
+}
+
+// TODO: Dummy type for ballot transaction. Should be remove when implement logic from core
+interface BallotTransactionData {
+  id: string;
+  address: string;
+  event: string;
+  category: string;
+  proposal: string;
+  network: string;
+  votedAt: string;
+  votingPower: string;
+}
+
+interface BallotTransation {
+  action: string;
+  actionText: string;
+  data: BallotTransactionData;
+  slot: string;
+  uri: string;
+  ownerUrl: string;
+  eventName: string;
 }
```

Comment:
Can we put these types inside another file, maybe create a type file for the Sign component.